File: Java/Desafios Iniciais - GFT Start #3 Java/2-Resto 2/solucao.java
/* Desafio
Leia um valor inteiro N. Apresente todos os números entre
 1 e 10000 que divididos por N dão resto igual a 2.

Entrada
A entrada contém um valor inteiro N (N < 10000).

Saída
Imprima todos valores que quando divididos por 
N dão resto = 2, um por linha.

 
Exemplo de Entrada	Exemplo de Saída
13                             2
                              15
                              28
                              41
                              ... */

import java.io.IOException;
import java.util.Scanner;

public class solucao {

  public static void main(String[] args) throws IOException {
    Scanner leitor = new Scanner(System.in);
    int N = leitor.nextInt();

    for (int i = 0; i < 10000; i++) {
      if (i % N == 2)
        System.out.println(i);
    }
  }
  // Complete com a sua lógica nos espaços em branco                                   
}

File: C-Sharp/Desafios avançados em C#/6-Visita na Feira/solucao.cs
/* Desafio
Você está na feira com a sua sacola e parou em uma banca.
 O feirante lhe entregou pimentões amarelos e vermelhos. 
 Agora iremos somar os pimetões amarelos e vermelhos
  para descobrir o total de pimentões na sacola. 
   Você receberá 2 inteiros que devem ser lidos 
   e armazenados nas variáveis A (pimentões amarelos)
    e B (pimentões vermelhos).
    Faça a soma de A e B atribuindo o seu 
    resultado na variável X (total de pimentões).
     Apresente X como descrito na mensagem de exemplo abaixo.
      Não apresente outra mensagem além da mensagem especificada.

Entrada
A entrada contém 2 valores inteiros, separados por um espaço.

Saída
Imprimir a mensagem "X = " (sendo a letra X maiúscula)
 seguido pelo valor da variável X e pelo final de linha. 
 Assegure que exista um espaço antes e depois do sinal de igualdade.

 
Exemplos de Entrada	Exemplos de Saída
11 7                    X = 18

-11 6                   X = -5

11 -2                   X = 9


 */



using System;

class solucao
{

  static void Main(string[] args)
  {

    int a, b, x;
    string[] entrada = Console.ReadLine().Split();
    a = Convert.ToInt32(entrada[0]);
    b = Convert.ToInt32(entrada[1]);

    x = a + b; //complete o código de acordo com o cálculo esperado

    Console.WriteLine("X = {0}", x);

  }

}